packages/teams-js/src/internal/constants.ts

```diff
@@ -137,12 +137,14 @@ export const validOrigins = [
   'outlook.office365.com',
   'outlook-sdf.office365.com',
   '*.teams.microsoft.com',
+  '*.www.office.com',
   'www.office.com',
   'word.office.com',
   'excel.office.com',
   'powerpoint.office.com',
   'www.officeppe.com',
-  '*.www.office.com',
+  '*.www.microsoft365.com',
+  'www.microsoft365.com',
```

Comment:
Are there really going to be subdomains that include the www part? E.g. foobar.www.microsoft.365.com instead of just foobar.microsoft365.com?

Reply:
I had the same question..

Reply:
Yes, our test environments will live at blah.www.microsoft365.com (not at blah.microsoft365.com). Thanks!